docs/csharp/nullable-references.md

```diff
@@ -57,21 +57,21 @@ The behavior is the same as previous versions of C#.
 
 The compiler uses the following rules in an enabled nullable context:
 
-- Any variable of a reference type is a **non-nullable reference type**.
+- Any variable of a reference type is a **non-nullable reference**.
 - Any non-nullable reference may be dereferenced safely.
-- Any nullable reference type (noted by `?` after the type in the variable declaration) may be null, and must be verified as not null before it can be dereferenced.
+- Any nullable reference type (noted by `?` after the type in the variable declaration) may be null. Static analysis determines if it the value is known to be not null when it is dereferenced. If not, the compiler warns you.
```

Comment:
There's some grammar hiccup in the "Static analysis determines if it the value is known..." sentence. Should "it" be removed?